func@0.23.rb

```diff
@@ -0,0 +1,47 @@
+require 'fileutils'
+
+class Func < Formula
```

Comment:
```suggestion
class FuncAT023 < Formula
```

Reply:
Ahh thank you!

Reply:
This reviews are hard to review inspect by eye, specially the shasum strings. 
There need to be a GitHub action that cares of this for every bump request in addition of adding a e2e test

Reply:
I agree with you Carlos